js/scenes/PotLuckInfoScreen/PotluckInfoScreenContainer.js

```diff
@@ -2,19 +2,29 @@ import React, { Component } from 'react';
 import { Button } from 'react-native';
 import PotluckInfoScreen from './PotluckInfoScreen';
 
+import CreatePotluckProgressBar from '../../components/CreatePotluckProgressBar';
+
+import NavigationArrow from '../../components/NavigationArrow';
+
 class PotluckInfoScreenContainer extends Component {
   static navigationOptions = {
     title: 'Info',
-    header: ({ navigate }) => ({
+    header: ({ navigate, goBack }) => ({
+      style: { height: 0, margin: 0, padding: 0, },
+      title: <CreatePotluckProgressBar title='Info' progressNumber={3} />,
```

Comment:
3 is a magic number here. Is there somewhere we could store these as named constants and import them?